rmw_connextdds_common/src/common/rmw_impl.cpp

```diff
@@ -219,6 +221,18 @@ rmw_connextdds_find_string_in_list(
   return false;
 }
 
+rmw_ret_t
+rmw_connextdds_duration_from_ros_time(
+  DDS_Duration_t * const duration,
+  const rmw_time_t * const ros_time)
```

Comment:
IMO, there's no reason to have a return code here, this function should never fail.

```suggestion
DDS_Duration_t
rmw_connextdds_duration_from_ros_time(
  const rmw_time_t * const ros_time)
```